magma/home/__init__.py

```diff
@@ -21,14 +21,26 @@
 @home_page.route('/', methods=['GET', 'POST'])
```

Comment:
I initially wrote it as two routes, but the GET route would only have the `return render_template('index.html')` line in it so I guess unless there's more logic that would need to go into the GET route I'm not sure it's simpler to have two routes.